### PR TITLE
calligra: restore libkokross.so

### PR DESCRIPTION
**Pull request by sgn**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/calligra/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ makedepends="akonadi-contacts-devel akonadi5-devel ecm-devel eigen3.2
  libopenexr-devel libspnav-devel libvisio-devel libwpg-devel libwps-devel
  poppler-qt5-devel qca-qt5-devel threadweaver-devel marble5-devel
  kauth-devel kparts-devel kcmutils-devel kdelibs4support-devel
- ktextwidgets-devel kross-devel kservice-devel kcrash-devel"
```
Comment (sgn) [inline srcpkgs/calligra/template:18]: `kross-devel` was specified twice.

File: common/shlibs [hunk 2]
```diff
@@ -2404,6 +2404,7 @@ libpinktrace_0.9.so.9 pinktrace-0.9.1_1
 libsyd_1.0.so.0 sydbox-1.0.2_1
 libcsync.so.0 csync-0.50.0_1
 libcalligrasheetsodf.so.17 calligra-sheets-3.2.0_1
+libkokross.so.17 calligra-libs-3.2.0_3
```
Comment (Johnnynator) [inline common/shlibs:2407]:
```suggestion
libkokross.so.17 calligra-libs-3.2.0_1
```
Just go with revision 1, there

File: srcpkgs/calligra/template [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'calligra'
 pkgname=calligra
 version=3.2.1
-revision=2
+revision=3
```
Comment (Johnnynator) [inline srcpkgs/calligra/template:4]: Why revbump? No packahge content would change?

**Review by Johnnynator — CHANGES_REQUESTED** [review]

File: common/shlibs (hunk 2 above)
Comment (sgn) [reply common/shlibs:2407]: No, `libkokross.so.17` isn't available in `_2`, I think some dependencies changed and it's available now.

File: srcpkgs/calligra/template (hunk 3 above)
Comment (sgn) [reply srcpkgs/calligra/template:4]: `libkokross.so.17` isn't available in `_2`,

File: common/shlibs (hunk 2 above)
Comment (Johnnynator) [reply common/shlibs:2407]: So it is available in 3.2.1_3 ? Not .0_ ?

Comment (sgn) [reply common/shlibs:2407]: Ah, yes, it's available in `3.2.1_3` not in `3.2.1_2` and `3.2.0`

**Review by Johnnynator — APPROVED** [review]

**Pull request merged**